src/pages/main-page/AboutUsSection.tsx

```diff
@@ -1,11 +1,35 @@
 import React from 'react';
 import styles from './assets/about-us-section.module.scss';
+import FirstPlanet from '../../icons/FirstPlanet';
+import SecondPlanet from '../../icons/SecondPlanet';
+import ThirdPlanet from '../../icons/ThirdPlanet';
 
 export default function AboutUsSection(props: any) {
     return (
         <div className={styles.sectionContainer}>
-            O NAS
-
+            <div className={styles.sectionWrapper}>
+                <div className={styles.sectionTitle}>
+                    {__('aboutUs.header')}
+                </div>
+                <div className={styles.sectionDivider}></div>
+                <div className={styles.sectionContent}>
+                    {__('aboutUs.text')}
+                </div>
+                <div className={styles.sectionPlanets}>
+                    <div className={styles.planetContainer}>
```

Comment:
I was thinking about extract it to component like (example):

`PlanetComponent(props) {`
`return (`
`<>`
                    `<div className={styles.planetContainer}>`
`{props.icon()}`
`{props.text}`
`</div>`
`<>`
`);`
`}`